zio-http/src/main/scala/zhttp/service/RequestBodyHandler.scala

```diff
@@ -0,0 +1,22 @@
+package zhttp.service
+import io.netty.channel.{ChannelHandlerContext, SimpleChannelInboundHandler}
+import io.netty.handler.codec.http.{HttpContent, LastHttpContent}
+import zhttp.http.HttpData.{UnsafeChannel, UnsafeContent}
+
+final class RequestBodyHandler(val callback: UnsafeChannel => UnsafeContent => Unit)
+    extends SimpleChannelInboundHandler[HttpContent](false) { self =>
+
+  private var onMessage: UnsafeContent => Unit = _
+
+  override def channelRead0(ctx: ChannelHandlerContext, msg: HttpContent): Unit = {
+    self.onMessage(new UnsafeContent(msg))
```

Comment:
Shouldn't we call `ctx.fireChannelRead(msg)` here ?

Reply:
This is not needed since this handler is listening to HttpContent and handling all of them